apps/govquests-frontend/src/domains/action_tracking/strategies/ActionStrategyFactory.ts

```diff
@@ -29,22 +24,10 @@ export class ActionStrategyFactory {
         return VerifyPositionStrategy;
       case "send_email":
         return SendEmailStrategy;
-      case "wallet_verification":
-        return VerifyWalletStrategy;
-      case "verify_delegate":
-        return VerifyDelegateStrategy;
-      case "verify_delegate_statement":
-        return VerifyDelegateStatement;
       case "verify_agora":
         return VerifyAgora;
-      case "verify_first_vote":
-        return VerifyFirstVote;
-      case "holds_op":
-        return HoldsOpStrategy;
-      case "become_delegator":
-        return BecomeDelegatorStrategy;
       default:
-        throw new Error(`Unsupported action type: ${actionType}`);
+        return DefaultStrategy;
```

Comment:
You're right, I've considered here that the default strategy would be the one used on Hold_op, verify delegate and other strategies. Do you think it would be better adding a "case" with those ones and throw an error by default?